contrib/tools/workflowcheck/determinism/determinism_test.go

```diff
@@ -43,7 +43,7 @@ func Test(t *testing.T) {
 		determinism.NewChecker(determinism.Config{
 			IdentRefs:         identRefs,
 			SkipFiles:         []*regexp.Regexp{regexp.MustCompile(`.*/should_skip\.go`)},
-			Debug:             true,
+			Debug:             false, // Set to true to see details
```

Comment:
This is unrelated to the rest of the issue, but is too noisy to see other test output